src/software/thunderscope/binary_context_managers.py

```diff
@@ -137,7 +139,7 @@ def __restart__(self):
 
             time.sleep(1)
 
-    def __exit__(self, type, value, traceback):
+    def __exit__(self, type: Any, value: Any, traceback: Any) -> None:
```

Comment:
I think its ok if we don't type hint dunder methods since their signatures can be looked up easily. For `__exit__`, looks like the proper type annotations [are these](https://stackoverflow.com/a/58808179). But let's just not use any annotations bc it looks messy

Reply:
does this include the __init__ methods as well or just __exit__? also, do you want me to remove the argument type annotations or just the return type annotations? thanks!

Reply:
Just remove the argument type annotations for `__exit__`. We should type hint `__init__` since we have our own params in there